README.md

```diff
@@ -322,6 +323,37 @@ cv2.imwrite('youtube_frame.jpg', frame)
 grabber.release()
 ```
 
+### File Stream
+Here's an example of using FrameGrab to capture frames from a video file:
+
+```python
+from framegrab import FrameGrabber
+import cv2
+
+config = {
+    'input_type': 'file_stream',
+    'id': {
+        'file_path': 'path/to/your/video.mjpeg'  # or .mp4, .avi, etc.
```

Comment:
FWIW, I tried it with a .mov, and it worked, so that's nice. Maybe we should mention that.